proposals/0281-visible-forall.rst

```diff
@@ -700,10 +750,10 @@ Type checking
 8. Generalize the ``ITVDQ`` rule introduced earlier
    by using ``t2t``::
 
-     rho = t2t(e)
-     G |- sigma[a := rho];         pis  ~>  Theta; phis; rho_r
-     ---------------------------------------------------------- ITVDQ-T2T
-     G |- (forall a -> sigma);  e, pis  ~>  Theta; phis; rho_r
+     sigma_a = t2t(e)
+     G |- sigma_b[a := sigma_a];         pis  ~>  Theta; phis; rho_r
+     --------------------------------------------------------------- ITVDQ-T2T
+     G |- (forall a -> sigma_b);      e, pis  ~>  Theta; phis; rho_r
 
    ``t2t`` transforms term arguments into type arguments, see "T2T-Mapping"
    below for an informal definition.
```

Comment:
We need a new point

9.  In the same way, generalise the typechecking rule for patterns, to invoke `p2t` to transform a pattern to a type, when the constructor has a visible forall in its type (see Section 7.6).

Reply:
I expect each point to stand alone, so if it starts with "in the same way", it can't be a separate point. I'll just add this sentence to point 8.